migrations/20220429162252-create-user.js

```diff
@@ -61,10 +61,6 @@ module.exports = {
         type: DataTypes.STRING,
         allowNull: true,
       },
```

Comment:
Now we don't need email?

Reply:
We have it on line 

> Good work @capitale250 , please address the following in addition to in-code change requests:
> 
> * The PR description is confusing, what does the PR mainly do? should be something like `chore (documentation & testing): assign-driver-too-bus`
> * the branch name is even more confusing, should indicate the main task the branch solves, jus like the PR title
> * It is best practice to separate concerns. Combining documentation and testing in one branch is something we should at best avoid next time
> * In code, I have only pointed out samples of bad practice, eg, commented-out code, console.log statements, etc. Please go on and remove every occurrence
> * Hooks are not test cases, only compare results in test cases
> * You have basically repeated one single test all through
> * The test quality is so in adequate, i think even the biggest bugs will just slip through. You are basically checking only the status code, that the response body is an object - which always is, and just that one property name, eg message exists in the response body. It will be good to test some specific properties and their values in the response body for different responses.
> * There are other review comments prior to these, that you also need to respond to. Addressing them and not responding in conversation hides visibility of the activity, please respond to all review comments within the PR (conversation)
> * The test suit as of now is not passing, please check this attached screenshot
> 
> <img alt="Screen Shot 2022-06-23 at 09 34 23" width="1440" src="https://user-images.githubusercontent.com/36167006/175244732-431ee700-c34b-4443-aaa0-952a8198503a.png">

did you run undo and do migrations and seeders